modules/swagger-generator/pom.xml

```diff
@@ -65,7 +65,7 @@
                     <scanIntervalSeconds>10</scanIntervalSeconds>
                     <stopPort>${JETTY_TEST_STOP_PORT}</stopPort>
                     <stopKey>STOP</stopKey>
-                    <useTestScope>true</useTestScope>
+                    <useTestScope>false</useTestScope>
```

Comment:
`jetty-maven-plugin` should be used only for testing, therefore it makes sense to have it use test scope, why do you need to change the scope?